pkg/utils/utils.go

```diff
@@ -211,3 +212,25 @@ However, you can access your serial using a web browser:
 	}
 	return nil
 }
+
+func SSHGetFingerprint(key string) (string, error) {
+	tmp, err := ioutil.TempFile("", ".tmp")
+	if err != nil {
+		return "", fmt.Errorf("Unable to create a tempory file: %v", err)
+	}
+	buff := []byte(key)
+	bytesWritten := 0
+	for bytesWritten < len(buff) {
+		nb, err := tmp.Write(buff[bytesWritten:])
+		if err != nil {
+			return "", fmt.Errorf("Unable to write: %v", err)
+		}
+		bytesWritten += nb
+	}
+	defer os.Remove(tmp.Name())
+	ret, err := exec.Command("ssh-keygen", "-l", "-f", tmp.Name()).Output()
```

Comment:
Create an issue for later: do not use external program to compute the fingerprint